k8s/kube-base/deployment-xdmod.yaml

```diff
@@ -42,6 +42,34 @@ spec:
               mountPath: /run/httpd
             - name: vol-php-session
               mountPath: /var/lib/php/session
+        - image: moc-xdmod:latest
+          name: xdmod-acl-work-a-round
+          imagePullPolicy: IfNotPresent
+          command: ["/usr/bin/bash"]
+          args: ["-c", "while true; do /usr/bin/acl-config; sleep 420; done"]
```

Comment:
What's here is absolutely fine, but consider combining `command` and `args` and using a literal block to make the code easier to read:

```
command:
  - /bin/bash
  - -c
  - |
    while true; do
      /usr/bin/acl-config
      sleep 420
    done
```